lrpar/examples/start_states/src/comment.y

```diff
@@ -1,4 +1,7 @@
-%grmtools{yacckind: Original(GenericParseTree)}
+%grmtools{
+    yacckind: Original(GenericParseTree),
+    test_files: "input.txt",
```

Comment:
Is this using the example also as a test?

Reply:
Sort of in the sense that it runs input.txt through the parser on build.rs, but not in the sense of being gated by `#[cfg(test)]`, that is also a bad thing though in that it is hard to control when cargo rebuilds build.rs